plugins/notifications/slack/slack_plugin.go

```diff
@@ -28,8 +29,19 @@ var logger hclog.Logger = hclog.New(&hclog.LoggerOptions{
 })
 
 func (n *Notify) Notify(ctx context.Context, notification *Notification) (*Empty, error) {
+	if _, ok := n.ConfigByName[notification.Name]; !ok {
+		return nil, fmt.Errorf("invalid plugin config name %s", notification.Name)
+	}
+	cfg := n.ConfigByName[notification.Name]
+	if cfg.LogLevel != nil {
```

Comment:
I think we want to have the same check in both plugins, so maybe check here for non-empty string ?